File: Python/Exercicio27.py
# Algoritmo que carrega um vetor com 10 números inteiros.
# Mostra o vetor na ordem inversa a que foi digitado.

# Variável
vetor = []

# Processamento
for i in range(1, 11):
    # Entrada
    valor = int(input("Digite um valor: "))
    vetor.append(valor)

# Saída
vetor.reverse() # Inverte a lista
for i in vetor:
    print(i)

File: C/Exercicio27.c
//Algoritmo que carrega um vetor com 10 números inteiros.
//Mostra o vetor na ordem inversa ao que foi digitado.
#include <stdio.h>

int main() {
    //Variáveis
    int vetor[10], i;

    for (i = 0; i < 10; i++) {
        //Entrada
        printf("Digite um valor: \n");
        scanf("%d", &vetor[i]);
    }

    //Saída
    printf("Ordem inversa: \n");
    for (i = 9; i >= 0; i--) {
        printf("%d \n", vetor[i]);
    }

    return 0;
}